EditPortfolio.css

```diff
@@ -0,0 +1,369 @@
+  color: #000000;
+  margin-top: 35px;
+  margin-bottom: 3px;
+}
+
+.skills {
+  font-family: "Fjalla One", sans-serif;
+  font-size: 18px;
+  color: #5d5151;
+  line-height: 15px;
+  margin-top: 10px;
+  align-self: center;
+}
+
+#container {
+  width: 75%;
+  padding: 20px;
+  margin-left: 10px;
+}
+.border {
+  border: 5px solid #ffffff;
+  border-width: 2px;
+  border-radius: 5px;
+  padding-top: 0px;
+  padding-left: 15px;
+  padding-right: 5px;
+  padding-bottom: 3px;
+  margin-bottom: 10px;
+}
+
+.title {
+  margin-top: 50px;
+
+  font-family: "Fjalla One", sans-serif;
+  font-size: 30px;
+  color: #000000;
+  margin-bottom: 10px;
+}
+.heading2 {
+  font-family: "Francois One";
+  font-size: 22px;
+  color: #555;
+  margin-top: 10px;
+  margin-bottom: 3px;
+}
+
+.paragraph {
+  color: #5d5151;
+  margin-top: 5px;
+  line-height: inherit;
+  justify-content: right;
+  padding-right: 20px;
+  text-align: justify;
+  font-size: 16px;
+}
+
+.linkedIn {
+  color: #5d5151;
+  margin-top: 5px;
+  font-size: 15px;
+  font-weight: bold;
+}
+.linkedIn a {
+  margin-left: 30px;
+  text-decoration: none;
+  color: #ffffff;
+}
+.linkedIn a :hover {
+  color: #d6ebba;
+}
+
+footer {
+  background-color: #333;
+  color: #fff;
+  padding: 10px 0;
+  width: 100%;
+}
+
+.addButton {
+  background-color: #17a2b8;
+  color: #fff;
+  border: none;
+  border-radius: 5px;
+  padding: 10px 20px;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+  margin-top: 0px;
+  margin-bottom: 10px;
+}
+.span {
+  color: rgb(38, 38, 39);
+  font-weight: bold;
+  margin-right: 10px;
+}
+
+.model {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+.model-content {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 50%;
+  margin-top: 40px;
+  height: auto;
+  transform: translate(-50%, -50%);
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 10px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+#formContainer {
+  background-color: #fff;
+}
+
+#formContainer label {
+  font-family: "Francois One", sans-serif;
+  letter-spacing: 0.5px;
+  font-size: 16px;
+  font-weight: bold;
+  color: #054282;
+
+  margin-top: 20px;
+  margin-bottom: 5px;
+}
+.formContainer h1 {
+  font-family: "Fjalla One";
+  text-align: center;
+  font-size: 18px;
+  margin-bottom: 0px;
+  font-weight: bold;
+  color: #054282;
+}
+
+#formContainer input[type="text"],
+#formContainer textarea {
+  width: 100%;
+  padding: 8px;
+  margin: 6px 0;
+  font-size: 17px;
+  font-family: "Fjalla One", sans-serif;
+  padding: 10px;
+  border-radius: 5px;
+  transition: border-color 0.2s ease;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px;
+  border: 1px solid #17a2b8;
+  border-width: 3px;
+  border-radius: 5px;
+  margin-bottom: 10px;
+  background-color: #fff;
+  box-shadow: 0 10px 8px rgba(99, 98, 98, 0.1);
+  cursor: pointer;
+  box-sizing: border-box;
+}
+
```

Comment:
There is too much duplication of css in formContainer. It would be better to extract the common css in one class and use that class where that style is needed.